packages/test-generator/integration-test-templates/cypress/e2e/form/DSAllSupportedFormFields-spec.cy.ts

```diff
@@ -155,9 +155,10 @@ describe('FormTests - DSAllSupportedFormFields', () => {
     });
   });
 
-  specify('update form should display current values and save to DataStore', () => {
+  specify('update form should display current values, update them, and save to DataStore', () => {
     cy.get('#DataStoreFormUpdateAllSupportedFormFields').within(() => {
-      // TODO: check current values on all fields and change
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(3000);
```

Comment:
Why was a wait added here?

Reply:
The tests were failing without the wait.

Reply:
Failing in what way, was the page not ready for the rest of the test to execute? Waiting for a fixed period of time is an anti-pattern, is there anything more specific that can be done instead like waiting for a specific element to be rendered?

Reply:
When I was first writing the tests, the page was not ready for the rest of the tests to execute. However, I just removed the `wait` and the tests executed successfully, so I'm happy to remove this line.